haiku-games/minesweeper/minesweeper-20150109.recipe

```diff
@@ -2,8 +2,11 @@ SUMMARY="A Windows Minesweeper clone"
 DESCRIPTION="Minesweeper is a clone of the original Windows minesweeper game. \
 It supports different board sizes and best time saving."
 HOMEPAGE="https://github.com/HaikuArchives/Minesweeper"
-SOURCE_URI="git+https://github.com/HaikuArchives/Minesweeper#702046e61588c953e759f7792930c5acc258359a"
-REVISION="3"
+REVISION="5"
+COMMIT="702046e61588c953e759f7792930c5acc258359a"
+SOURCE_URI="$HOMEPAGE/archive/$COMMIT.tar.gz"
+CHECKSUM_SHA256="f532980a4f507375a982c2efcc09118e5a8cfedb9fd0775563ed236c2a70304f"
+SOURCE_DIR="minesweeper-$COMMIT"
```

Comment:
I would replace
```
REVISION="5"
COMMIT="702046e61588c953e759f7792930c5acc258359a"
SOURCE_URI="$HOMEPAGE/archive/$COMMIT.tar.gz"
CHECKSUM_SHA256="f532980a4f507375a982c2efcc09118e5a8cfedb9fd0775563ed236c2a70304f"
SOURCE_DIR="minesweeper-$COMMIT"
```
by
```
REVISION="4"
srcGitRev="702046e61588c953e759f7792930c5acc258359a"
SOURCE_URI="$HOMEPAGE/archive/$srcGitRev.tar.gz"
CHECKSUM_SHA256="f532980a4f507375a982c2efcc09118e5a8cfedb9fd0775563ed236c2a70304f"
SOURCE_DIR="minesweeper-$srcGitRev"
```
That's because srcGitRev is imho a slightly better choice than COMMIT, since:
* there are already 36 recipes using "srcGitRev", compared to only 4 which are using "COMMIT";
* contrary to "COMMIT", which looks like a reserved HaikuPorter variable, "srcGitRev" makes it clear that it is not a reserved but a custom variable.

And since the resulting SOURCE_URI is unchanged we don't really need to bump REVISION.

But if another contributor says that COMMIT is ok and/or that we should bump REVISION then that's OK for me too ;-)

Can you please also move COPYRIGHT and LICENSE after HOMEPAGE (and before REVISION?
(The correct order is given in our [HaikuPorter Guidelines](https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#ordering)).